### ISSUE TITLE
Blacklisted if possible

### ISSUE THREAD

**Issue by SatisfactoryBug** [Ephellon/Twitch-Tools#4]
Great idea but would you be able to add Blacklisted as Unwanted Twitch don't not always work as still showing

**Comment by Ephellon** [comment]
I'm sorry, but it is hard to follow your sentence. Did you mean: *Add a blacklist feature to prevent unwanted channels from appearing*?

**Comment by SatisfactoryBug** [comment]
Yeah this is what i mean and sorry for not understanding clearer

**Comment by Ephellon** [comment]
No problem. I'll add it to the **Experimental Features** section for `v5.13`

**Comment by SatisfactoryBug** [comment]
That brilliant also Hope it not too much and hope it not too much hassle if could add Theatre mode on/Off so any page open will open in Theatre mode or i do that in the new page Issues [it not really a issues just a idea]

**Comment by Ephellon** [comment]
Do you mean the [View Mode](https://github.com/Ephellon/Twitch-Tools/wiki/View-Mode) option?

**Comment by SatisfactoryBug** [comment]
Thank for that as yes i go change it the View Mode option again Thank

**Comment by SatisfactoryBug** [comment]
:ast one and that it, after the  Experimental Features section for v5.13 will it include the Backup setting so export / import as that way do not need to repeating fill-in  unwanted channels from appearing

Have a nice weekend thank again

**Comment by Ephellon** [comment]
That is currently under the **Export Settings** option

**Comment by Ephellon** [comment]
![image](https://user-images.githubusercontent.com/8632548/166151245-f1c2d37e-7eed-474b-83c3-0a9a4989b7d8.png)

**Comment by SatisfactoryBug** [comment]
Ohhh in the Networking Opps as i did look for Backup, i leave you alone and thank......

**Comment by Ephellon** [comment]
No problem

**Comment by SatisfactoryBug** [comment]
Sorry, I click Download but nothing happen i mean no popup like saved to Desktop etc nothing so i thinking where did the export file saved ?

**Comment by Ephellon** [comment]
If you save your setting sand click **Upload** it is saved to the internet. You can then **Download** the settings to another computer using the same code

**Comment by SatisfactoryBug** [comment]
So IF i click Upload then Format to Reinstall Windows 10 after that then click Download with the code it work ??

**Comment by Ephellon** [comment]
Yes

**Comment by Ephellon** [comment]
Just write the code down somewhere

**Comment by SatisfactoryBug** [comment]
Brillant, Thank You very much

**Comment by Ephellon** [comment]
Twitch has a built-in `/block [username]` feature available in any chat

**Comment by SatisfactoryBug** [comment]
Was hoping it worked but nope don't work as it still changing channel when someone finish streaming the channel still change to the other steam that don't block, was worth a try

**Comment by Ephellon** [comment]
I'm unsure what you're trying to accomplish. What exactly are you trying to get done?
